iidm/iidm-geodata/src/main/java/com/powsybl/iidm/geodata/odre/OdreGeoDataAdderPostProcessor.java

```diff
@@ -56,7 +56,7 @@ public OdreGeoDataAdderPostProcessor(Path substationsFilePath, Path aerialLinesF
 
     private static Path getEquipmentFileFromConfig(PlatformConfig platformConfig, String type) {
         Objects.requireNonNull(platformConfig);
-        return platformConfig.getOptionalModuleConfig("odre-geodata-post-processor")
+        return platformConfig.getOptionalModuleConfig("odre-geodata-data-importer-post-processor")
```

Comment:
The documentation states that the module is named `odre-geo-data-importer-post-processor`.
I think that the documentation name is better than the other.